packages/captp/lib/captp.js

```diff
@@ -103,8 +138,13 @@ export function makeCapTP(ourId, send, bootstrapObj = undefined) {
     return harden(pr);
   }
 
+  // Set up import
   function convertSlotToVal(theirSlot) {
     let val;
+    // Invert slot direction from other side.
+    // (??? cwebber): Why is this inverted this way?  In what circumstance
```

Comment:
The direction prevents namespace collisions between the slots we allocate and the ones that the other side allocates.  If we allocate a slot, serialise it to the other side, and they send it back to us, we need to reference just our own slot, not one from their side.